iothub/device/src/PlugAndPlayConvention/PnpConventionHelper.cs

```diff
@@ -50,6 +49,20 @@ public static Message CreateIothubMessageUtf8(string telemetryName, string seria
         /// <summary>
         /// Create a key-value property patch for both read-only and read-write properties.
         /// </summary>
+        /// <remarks>
+        /// For a root-level property update, the patch is in the format:
+        ///     {
+        ///         "samplePropertyName": 20
+        ///     }
+        ///
+        /// For a component-level property update, the patch is in the format:
+        ///     {
+        ///         "sampleComponentName": {
+        ///             "__t": "c",
+        ///             "samplePropertyName"": 20
+        ///         }
+        ///     }
+        /// </remarks>
         /// <param name="propertyName">The property name, as defined in the DTDL interface.</param>
         /// <param name="serializedPropertyValue">The serialized property value, in the format defined in the DTDL interface.</param>
         /// <param name="componentName">(optional) The name of the component in which the property is defined. Can be null for property defined under the root interface.</param>
```

Comment:
I don't love the ternary operator for big if/else. For readability, I think it'll look better to use an if/else block.